app/src/androidTest/kotlin/com/wire/android/feature/auth/registration/CreateAccountActivityUITest.kt

```diff
@@ -22,5 +22,10 @@ class CreateAccountActivityUITest : FunctionalTest(CreateAccountActivity::class.
         onView(withId(R.id.createProAccountTitleTextView)).check(matches(isDisplayed()))
         onView(withId(R.id.createProAccountDescriptionTextView)).check(matches(isDisplayed()))
     }
+
+    @Test
+    fun launch_uiElementsVisibleInLandscape() = inRotatedMode {
```

Comment:
I would continue using ```camelCase``` for consistency. Any other reason for the usage of ```_```?

Reply:
It actually adds a layer between what's being tested and the expected outcome. Kind of like a `givenThis_whenThat_thenCheckThose()` thing. Except that we don't have given in this case (Given that app is installed?). So, it's like (when screen is launched)_(check that ui elements are visible). This is fine btw, according to Kotlin coding style guidelines: https://kotlinlang.org/docs/reference/coding-conventions.html#function-names

Reply:
Fair enough.